.github/workflows/multichain-prod.yml

```diff
@@ -14,6 +14,12 @@ on:
         required: true
       tag:
         description: Image tag that you wish to deploy, either by SHA or Version/latest
+      node_group:
```

Comment:
I think this is not a prod env, let's name files accordingly. And add "deploy" here and in dev.